dts/msm8226/msm8926-samsung-r02.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0xC808FF01 2 0x10001>;
+
+	matisselte {
+		model = "Samsung Galaxy Tab 4 10.1 Rev 2 (SM-T535)";
```

Comment:
Not sure if the `Rev 2` has any value here, given that we usually just support the latest revision (which is usually the only one on the market).

```suggestion
		model = "Samsung Galaxy Tab 4 10.1 (SM-T535)";
```

please change it for matissewifi as well.

Reply:
Done.